### ISSUE TITLE
Add Delight

### ISSUE THREAD

**Issue by C3nE3s** [C3nE3s/jamstack-101#7]
If time, add animations & delight to the page

*IMPORTANT*  - All animations should be behind reduced motion preference for accessibility 

- [x] Look if [this](https://github.com/jamiebuilds/tailwindcss-animate) tailwind plugin can add value and velocity to implementation
- [x] onEnter opacity and translate Y effects on graphical elements
- [ ] falling element section (make the brand logos appear to vertically fall)
- [ ] Possible typing effect on Header Elements onEnter?
- [ ] Heart Animation in footer
- [ ] Animate Mouse to follow path of SVG on scroll
